install.sh

```diff
@@ -145,10 +146,16 @@ __am_prompt_install() {
     local PROMPT_SHELL=$(echo $PROMPT_SHELL | tr '[:upper:]' '[:lower:]')
 
     # use the correct shell
-    . $AM_PROMPT/sh/scripts/use-shell $PROMPT_SHELL
+    . "$AM_PROMPT/sh/scripts/use-shell" $PROMPT_SHELL
 
     # open the changelog url
-    . $AM_PROMPT/sh/scripts/open-url $PROMPT_CHANGELOG_URL
+    . "$AM_PROMPT/sh/scripts/open-url" "https://github.com/automotivemastermind/prompt/blob/$PROMPT_SHA/CHANGELOG.md"
```

Comment:
is open-url always available? I remember we had an issue with this b4 but I can't remember what the problem

Reply:
I resolved that in a previous commit -- it will now print the URL to open in the event that none of the open cmds are available.

Reply:
for reference: https://github.com/automotiveMastermind/prompt/blob/2aaec39ecdc3de0df60b08d62d66d539a30f3034/src/sh/scripts/open-url#L13-L29